xline/src/storage/lease_store/mod.rs

```diff
@@ -52,6 +54,10 @@ where
     kv_update_tx: mpsc::Sender<(i64, Vec<Event>)>,
     /// Primary flag
     is_primary: AtomicBool,
+    /// cache unsynced lease id
+    unsynced_cache: Arc<RwLock<HashSet<i64>>>,
```

Comment:
If we don't need to hold the lock during await, please use `parking_lot::RwLock`

Reply:
done